src/components/mainContent/Navbar.js

```diff
@@ -18,7 +20,19 @@ export default function NavBar(props) {
       <button className="button" onClick={dropdownHandler}>
         <div className={`triangle ${dropdownOpen ? "up" : ""}`}></div>
         {dropdownOpen && (
```

Comment:
The triangle is now broken - not entirely sure what's happening here, to be honest.

Also a TODO for later - we should add an `aria-label` that explains what the button does.

Reply:
Is the issue through when GitHub is running it? When I do `npm start` on my end it works fine (as in it opens and closes on click). Jiin wrote the button section and a placeholder but I added replaced the placeholder with the actual divs and links.

I added in `aria-label` after `<button className="button"` written as `aria-label="Navigation Bar Button"`, is that alright?

Reply:
Ooh so I could've been more clear, by "broken" I mean the triangle moves different on click. Compare the versions at:

https://deploy-preview-9--infallible-mirzakhani-e36a0d.netlify.app/

https://infallible-mirzakhani-e36a0d.netlify.app/

Reply:
Oh I see, I wasn't aware of the previous version. I thought I only changed the dropdown content, I'm not sure if it's related to another property, maybe? I don't know which one it would be though.

Reply:
Hm, I'm not entirely sure what's causing it either, but I'm pretty adverse to implementing a regression in our code - do you mind looking into this a bit more? I assume it has to do with how we've changed the navbar styling.

Reply:
I fixed the issue! It was something to do with the display value for .dropdown, so I removed it and I believe the arrow works as before.